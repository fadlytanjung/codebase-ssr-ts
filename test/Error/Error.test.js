import React from 'react';
import { render } from '@testing-library/react';
import Error from '@/pages/Error';

describe('Test Home Page', () => {
  test('It should render with snapshot', async () => {
    const { asFragment } = render(<Error />);
    expect(asFragment()).toMatchSnapshot();
  });
});
