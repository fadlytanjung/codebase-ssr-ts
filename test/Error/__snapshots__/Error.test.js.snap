// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Home Page It should render with snapshot 1`] = `
<DocumentFragment>
  <h1>
     Error Page 
  </h1>
</DocumentFragment>
`;
